locksmith/src/controllers/v2/verifierController.ts

```diff
@@ -156,4 +212,37 @@ export class VerifierController {
       })
     }
   }
+
+  // check is address is a Verifier of a specific lock
+  async isVerifierEnabled(request: Request, response: Response) {
```

Comment:
This is good, and we will need to make sure that the `request.params.verifierAddress` matches the token on the API request too!